packages/babel-plugin-proposal-unicode-property-regex/src/index.js

```diff
@@ -1,18 +1,22 @@
 import rewritePattern from "regexpu-core";
 import * as regex from "@babel/helper-regex";
 
-export default function() {
+export default function(api, options) {
+  const { useUnicodeFlag = true } = options;
```

Comment:
@mathiasbynens I know we pulled in this plugin, so I'm curious for your feedback. We usually try to keep our plugins focused on single tasks, and because this was defaulting to `false`, it was converting unicode to non-unicode regexes at the same time even though this plugin is centered around the property regex functionality.

Given that people can always enable `transform-unicode-regex` alongside this proposal, do you think it is worth having this flag to begin with? Wondering if we should just remove it entirely.

Reply:
This is reflected in the readme as well, I remember talking about this before (since yeah we do already have a plugin that does the es6 transform and ideally all of it will be in preset-env anyway